dask/dataframe/tests/test_dataframe.py

```diff
@@ -1131,6 +1132,12 @@ def test_random_partitions():
     assert len(a.compute()) + len(b.compute()) == len(full)
 
 
+def test_series_round():
+    ps = pd.Series([1.123, 2.123, 3.123, 1.234, 2.234, 3.234], name='a')
+    s = dd.from_pandas(ps, npartitions=3)
+    assert eq(s.round(), ps.round())
```

Comment:
As in the actual rounding code? I check for equality with a specified number of digits in the test on line 1139